### DISCUSSION TITLE
Tracking of the Ride

### DISCUSSION THREAD

**Discussion by RitikaHanish** [ONDC-Official/mobility-specification#111] [General]
@92shreyansh 
Should we enforce GET api URL in track_url of on_track api whose response will be lat lng of driver's location at protocol level ?
It needs to be standardized for inter-operability to work.

**Comment by NikithShetty** [comment]
The protocol doesn't specify what the response body structure looks like when calling the tracking url. We need to standardize the response body format.

Proposing to use JSON response format to send driver's current lat,lon

**Comment by saurabhTriffy** [comment]
We can use the on_track api with standard response format as well. A rate limit on the api will also be appreciated
